src/App.js

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
-import { HomePage } from './pages/HomePage';
-import './style/main.scss';
-import Layout from 'atomic-layout';
-import { About } from './About';
-import { Shop } from './Shop';
-import { Header } from './components/Header';
+import React from 'react'
+import { HomePage } from './pages/HomePage'
+import './style/main.scss'
+import Layout from 'atomic-layout'
+import { About } from './About'
+import { Shop } from './Shop'
```

Comment:
You export some pages like `pages/HomePage`, but some like `./About`. Maybe move all pages into a single parent folder?

```
- pages
  - HomePage
  - About
  - Shop
  ...
```